src/app/index.ts

```diff
@@ -1,8 +1,10 @@
 import { Hono } from "hono";
 import pingRoute from "../routes/ping.route";
+import eventsRoute from "../routes/events.route";
 
 const app = new Hono();
 
 app.route("/", pingRoute);
+app.route("/events", eventsRoute);
```

Comment:
this structure looks good, maybe add a prefix to ping as well because the "/" prefix could cover everything